@xen-orchestra/log/capture.test.js

```diff
@@ -0,0 +1,225 @@
+'use strict'
+
+const { describe, it, beforeEach } = require('node:test')
+const assert = require('node:assert/strict')
+const { createLogger } = require('@xen-orchestra/log')
+const { configure } = require('@xen-orchestra/log/configure')
+const { captureLogs } = require('@xen-orchestra/log/capture')
+const { createCaptureTransport } = require('@xen-orchestra/log/capture')
+const { dedupe } = require('@xen-orchestra/log/dedupe')
+
+describe('capture log', () => {
+  const logger = createLogger('test-logger')
+
+  const logsTransportDefault = []
+  const transportTest = message => {
+    logsTransportDefault.push(message)
+  }
+
+  const logsTransportFatal = []
+  const transportFatal = message => {
+    logsTransportFatal.push(message)
+  }
+
+  beforeEach(() => {
+    configure(
+      createCaptureTransport([
+        {
+          level: 'fatal',
+          transport: transportFatal,
+        },
+        {
+          level: 'debug',
+          transport: transportTest,
+        },
```

Comment:
I don't understand why are testing that, filtering on the level is outside of `captureLogs` scope :slightly_smiling_face:

Reply:
I tested what was in the Usage.md, there it was the "producer" and "consumer" part presenting the logger and the filter on the configure(). :-)